### PR TITLE
Use pure JS unzip to not require unzip CLI

### LINKED ISSUE #7: Post-install fails on windows because unzip missing
See https://github.com/breach/thrust/issues/227

```
events.js:72
throw er; // Unhandled 'error' event
^
Error: spawn ENOENT
at errnoException (child_process.js:1001:11)
at Process.ChildProcess._handle.onexit (child_process.js:792:34)
npm ERR! node-thrust@0.7.5 postinstall: node scripts/bootstrap.js
npm ERR! Exit status 8
```

### PR DESCRIPTION
**Pull request by morganrallen**
Fixes post install problem on Windows.

Fixes #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,11 +9,12 @@
   "version": "0.7.5",
   "dependencies": {
     "async": "0.9.x",
+    "fs-extra": "0.12.x",
+    "minimist": "1.1.x",
```
Comment (spolu) [inline package.json:13]: do you need those?

**Comment by spolu** [comment]
Looks good otherwise, I'll test it later today and will merge right after!

File: package.json (hunk 1 above)
Comment (morganrallen) [inline package.json:13]: I think those came in via merge master? They exist already... I think...

Comment (morganrallen) [inline package.json:13]: Yeah, that exists in master, only being show here because it's a merge I guess....

Comment (spolu) [inline package.json:13]: Alright!

**Comment by spolu** [comment]
This command does not preserve permissions (+x) on linux and OSX. We might want to make it platform specific for windows?

**Pull request merged**